fineract-core/src/main/java/org/apache/fineract/infrastructure/core/serialization/GoogleGsonSerializerHelper.java

```diff
@@ -105,7 +105,7 @@ public static GsonBuilder createGsonBuilder() {
     public static void registerTypeAdapters(final GsonBuilder builder) {
         builder.registerTypeAdapter(java.util.Date.class, new DateAdapter());
         builder.registerTypeAdapter(LocalDate.class, new LocalDateAdapter());
-        // NOTE: was missing, necessary for GSON serialization with JDK 17's restrictive module access
+        // NOTE: was missing, necessary for GSON serialization with JDK 21's restrictive module access
```

Comment:
Is this true? Can't say I know either way, I'm just wondering if this was an unintended search-and-replace change

Reply:
I have notice the behaviour since JDK 17 exception thrown is "unnamed module"